### PR TITLE
Chore merge from 436de31

### PR DESCRIPTION
**Pull request by julien-deramond**
- [x] https://github.com/twbs/bootstrap/commit/436de310ba3257e356ec25c09768745baca0d4bb: integrated as is even if the sentences are not entirely true since our "subtle" versions have the same colors for backgrounds for example. But all these messages and stuff will be revamped before releasing the v5.3 with or without the Orange dark mode.
- [x] https://github.com/twbs/bootstrap/commit/eec501552016064056c379fb9ae42e93225e6046 
- [x] https://github.com/twbs/bootstrap/commit/20ab8219a0caca784483e93083cde269055d0f3a: linked to https://github.com/twbs/bootstrap/pull/38175 for consistency
- [x] https://github.com/twbs/bootstrap/commit/e8b7d382767dcd73916ecd339c237fb49b9f3561 
- [x] https://github.com/twbs/bootstrap/commit/5dc7f4c165c39b63eb5541a66d5ad443e35aaf09
- [x] https://github.com/twbs/bootstrap/commit/16aff741ecf47b2ce682b625c1a6ab3ae80f8256
- [x] https://github.com/twbs/bootstrap/commit/1369b81492a5368ac40dbfac3d6553e32065f706
- [x] https://github.com/twbs/bootstrap/commit/bada2b9a90be728da0157a6f96e4de5364035f5f
- [x] https://github.com/twbs/bootstrap/commit/d9d2b4fdb25395237ee70ec6ad502280e13a4639
- [x] https://github.com/twbs/bootstrap/commit/cfd3560c6df1decc0d203041c1218d9ca2547a67
- [x] https://github.com/twbs/bootstrap/commit/15744ee1d04bcca03155c3bb37ee7e65a7b011c6
  - **Warning** ⚠️ : please double-check this one in detail, might have some regressions
  - Linked to https://github.com/twbs/bootstrap/pull/38176
- [x] https://github.com/twbs/bootstrap/commit/1299b163c7526b0e6892856d87e8710bc195d7f5
- ~~https://github.com/twbs/bootstrap/commit/03983e5b998d0fc34b8ff98394f541063420097a~~: nothing to do since we don't have any floating labels in Boosted
- [x] https://github.com/twbs/bootstrap/commit/cf6972ccf1f995394bf1d25b557bd43023b87f73
- [x] https://github.com/twbs/bootstrap/commit/82391bc7b7c41e40f6223b7cfac48c5c3ae56255
- [x] https://github.com/twbs/bootstrap/commit/3635f2cbb4e44f229ef00bbeec209ac8cd5a0f3d
- [x] https://github.com/twbs/bootstrap/commit/e69b618a1a5328744963bb7344aa16c7b0dd7c82
- [x] https://github.com/twbs/bootstrap/commit/06f89276958ffdd9dc00e0d00b7ede3e00aec5df
- [x] https://github.com/twbs/bootstrap/commit/a6b39f59ffd467283b8c525aa3289f11fe16ad68
- ~~https://github.com/twbs/bootstrap/commit/d0053937fdbcae3011725b9fb20dae433cd44ffd~~: nothing to do cause already merged vi Dependabot
- [x] https://github.com/twbs/bootstrap/commit/1a4b18cb140a52635d4351ea499df338885d5a38
- [x] https://github.com/twbs/bootstrap/commit/d9037ba77f2cc43139659056ac55387ad876854d

### CODE REVIEW

File: site/content/docs/5.3/migration.md [hunk 1]
```diff
@@ -257,10 +259,10 @@ Learn more by reading the new [color modes documentation]({{< docsref "/customiz
       <li><code>--bs-card-title-color</code></li>
       <li><code>--bs-danger-bg-subtle</code></li>
       <li><code>--bs-danger-border-subtle</code></li>
-      <li><code>--bs-danger-text</code></li>
+      <li><code>--bs-danger-text-emphasis</code></li>
       <li><code>--bs-dark-bg-subtle</code></li>
       <li><code>--bs-dark-border-subtle</code></li>
-      <li><code>--bs-dark-text</code></li>
+      <li><code>--bs-dark-text-emphasis</code></li>
```
[the remark this answers is not in the document]
Comment (julien-deramond) [reply site/content/docs/5.3/migration.md:265]: Oh yes you're right, forgot to change them. Fixed in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1872/commits/b5e80e6c2cbdd5f2f5f2edc1dbd18c846541dea4.

**Comment by julien-deramond** [comment]
> Sass variables inside buttons.md doesn't have the right title level on our side.

Good catch. Fixed in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1872/commits/6e6df267795d5dbf063892bec51367043fb53b9a

File: site/layouts/shortcodes/scss-docs.html [hunk 2]
```diff
@@ -39,5 +39,18 @@
     {{- $match = replace $match " !default" "" -}}
   {{- end -}}
 
-  {{- highlight $match "scss" "" -}}
+  <div class="bd-example-snippet bd-code-snippet bd-scss-docs">
+    <!--Boosted mod: added `border-light border-1 border-top-0`-->
+    <div class="d-flex align-items-center highlight-toolbar ps-3 pe-2 py-1 border-bottom border-light border-1 border-top-0">
+      <a class="font-monospace text-body-secondary text-decoration-none small" href="{{ .Site.Params.repo }}/blob/v{{ .Site.Params.current_version }}/{{ $file | replaceRE `\\` "/" }}">
```
[the remark this answers is not in the document]
Comment (julien-deramond) [reply site/layouts/shortcodes/scss-docs.html:45]: Wow I hadn't even paid attention that it was clickable. So yeah definitely something to do in terms of design.

**Review by louismaximepiton — APPROVED** [review]

File: site/layouts/shortcodes/scss-docs.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (louismaximepiton) [reply site/layouts/shortcodes/scss-docs.html:45]: See https://github.com/twbs/bootstrap/pull/38197

File: site/content/docs/5.3/utilities/colors.md [hunk 3]
```diff
@@ -132,10 +144,16 @@ RGB colors are generated from a separate Sass map:
...
 
 {{< scss-docs name="utilities-text-colors" file="scss/_maps.scss" >}}
 
+Color mode adaptive text colors are also available as a Sass map:
+
+{{< scss-docs name="theme-text-map" file="scss/_maps.scss" >}}
+
+{{< scss-docs name="theme-text-dark-map" file="scss/_maps.scss" >}}
+
 ### Utilities API
```
[the remark this answers is not in the document]
Comment (louismaximepiton) [reply site/content/docs/5.3/utilities/colors.md:157]: See https://github.com/twbs/bootstrap/pull/38175

**Pull request merged**